Docker build breaks when snap repo path contains spaces

Working directory:  
`/home/Lin-Buo-Ren/工作空間/個人專案/Snapcrafters Template Plus`

Error message:

```
Building desktop-gtk3 
snapcraft is running as a snap False, SNAP_NAME set to ''
make FLAVOR=gtk3 -j4
gcc -Wall -O2 -o bindtextdomain.so -fPIC -shared ./../src/bindtextdomain.c -ldl
make FLAVOR=gtk3 install DESTDIR=/home/Lin-Buo-Ren/工作空間/個人專案/Snapcrafters Template Plus/parts/desktop-gtk3/install
install -D -m755 desktop-launch /home/Lin-Buo-Ren/工作空間/個人專案/Snapcrafters Template Plus/parts/desktop-gtk3/install/bin/desktop-launch
install: target 'Plus/parts/desktop-gtk3/install/bin/desktop-launch' is not a directory
Makefile:28: recipe for target 'install' failed
make: *** [install] Error 1
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snapcraft/internal/common.py", line 67, in _run
    return runner(['/bin/sh'], stdin=run_file, **kwargs)
  File "/usr/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/sh']' returned non-zero exit status 2
```

--- a/glib-only/Makefile
+++ b/glib-only/Makefile
@@ -17,4 +17,4 @@ $(DEST_LAUNCHER):
 	@cat $(SRC_DIR)/mark-and-exec >> $(DEST_LAUNCHER)
 		
 install: $(DEST_LAUNCHER)
-	install -D -m755 $(DEST_LAUNCHER) $(DESTDIR)/bin/$(DEST_LAUNCHER)
+	install -D -m755 $(DEST_LAUNCHER) "$(DESTDIR)"/bin/$(DEST_LAUNCHER)

--- a/gtk/Makefile
+++ b/gtk/Makefile
@@ -25,6 +25,6 @@ $(DEST_LAUNCHER):
 	gcc -Wall -O2 -o $(BINDTEXTDOMAIN) -fPIC -shared $(SRC_DIR)/../src/bindtextdomain.c -ldl
 		
 install: $(DEST_LAUNCHER)
-	install -D -m755 $(DEST_LAUNCHER) $(DESTDIR)/bin/$(DEST_LAUNCHER)
-	install -D -m644 $(FLAVOR_FILE) $(DESTDIR)/
-	install -D -m644 $(BINDTEXTDOMAIN) $(DESTDIR)/lib/$(BINDTEXTDOMAIN)
+	install -D -m755 $(DEST_LAUNCHER) "$(DESTDIR)"/bin/$(DEST_LAUNCHER)
+	install -D -m644 $(FLAVOR_FILE) "$(DESTDIR)"/
+	install -D -m644 $(BINDTEXTDOMAIN) "$(DESTDIR)"/lib/$(BINDTEXTDOMAIN)

--- a/qt/Makefile
+++ b/qt/Makefile
@@ -40,10 +40,10 @@ desktop-launch:
 	@echo "USE_$(FLAVOR)=true" >> $(FLAVOR_FILE)
 		
 install: desktop-launch
-	install -D -m755 $(DEST_LAUNCHER) $(DESTDIR)/bin/$(DEST_LAUNCHER)
+	install -D -m755 $(DEST_LAUNCHER) "$(DESTDIR)"/bin/$(DEST_LAUNCHER)
 	install -D -m644 snappy-qt5.conf \
-		$(DESTDIR)/etc/xdg/qtchooser/snappy-qt5.conf
+		"$(DESTDIR)"/etc/xdg/qtchooser/snappy-qt5.conf
 	install -D -m644 snappy-qt4.conf \
-		$(DESTDIR)/etc/xdg/qtchooser/snappy-qt4.conf
-	install -D -m644 $(FLAVOR_FILE) $(DESTDIR)/
+		"$(DESTDIR)"/etc/xdg/qtchooser/snappy-qt4.conf
+	install -D -m644 $(FLAVOR_FILE) "$(DESTDIR)"/
 

